popup/translations.ts

```diff
@@ -57,13 +70,14 @@ export const createTextSetter = (
 
   setterHooks.push(hook)
 
-  return (...args) => {
+  return (...args: any[]) => {
     hook.args = args
+    // @ts-ignore
```

Comment:
you should be able to remove this now that the `property` is safely a member of `element`

Reply:
![image](https://user-images.githubusercontent.com/19150229/112319567-8b7a7f00-8cf1-11eb-820a-33ec1c98fe48.png)

Reply:
hmm, the element should be then typed as `HTMLElement` as well

Reply:
It still complains about read-only properties
![image](https://user-images.githubusercontent.com/19150229/112320109-0f346b80-8cf2-11eb-855e-fc55a4420c5e.png)

Reply:
ahh, damn. I'm pretty sure there's a TypeScript trick that can be used here. let me take a quick look.

Reply:
https://stackoverflow.com/questions/49579094/typescript-conditional-types-filter-out-readonly-properties-pick-only-requir

The solution is quite complex and maybe not worth tackling right now. Let's just keep the `@ts-ignore` there for now.

Reply:
Alternate, quick solution: just type the property with this:
```
type TranslatableProperties = 'innerHTML'

property: TranslatableProperties
```

this way we can extend the attributes there easily, and it's type safe.